src/beautifiers/exfmt.coffee

```diff
@@ -0,0 +1,36 @@
+###
+Requires https://github.com/lpil/exfmt
+###
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Exfmt extends Beautifier
+  name: "exfmt"
+  link: "https://github.com/lpil/exfmt"
+  executables: [
+    {
+      name: "exfmt"
+      cmd: "mix"
+      homepage: "https://github.com/lpil/exfmt"
+      installation: "https://github.com/lpil/exfmt"
+      additionalHelp: "If Elixir's mix is installed you can confirm exfmt is installed by running mix exfmt or install it with 'mix archive.install github lpil/exfmt tag v0.4.0'"
+      version: {
+        args: ['help','exfmt'],
+        parse: (text) -> text.match(/exfmt-(\d+\.\d+\.\d+)/)[1]
+      }
+      docker: {
+        image: "unibeautify/exfmt"
```

Comment:
This Docker image can be changed to use Mix 1.6 which supports `format`.
https://hexdocs.pm/mix/Mix.Tasks.Format.html

/cc @szeck87

Reply:
I think this Docker image has been removed. Should use official Docker image.
cc @szeck87